teamspeak/content.md

```diff
@@ -0,0 +1,131 @@
+# What is TeamSpeak?
+
+TeamSpeak 3 offers the ideal voice communication for online gaming, education and training, internal business communication, and staying in touch with friends and family. Our primary focus is delivering a solution that is easy to use, with high security standards, excellent voice quality, and low system and bandwidth usage.
+
+> [teamspeak.com](https://teamspeak.com/)
+
+%%LOGO%%
+
+# How to use this image
+
+```console
+$ docker run -p 9987:9987/udp -p 10011:10011 -p 30033:30033 -e TS3SERVER_LICENSE=accept %%IMAGE%%
```

Comment:
Do you think it'd be good to encourage users to do `TS3SERVER_LICENSE=view` before they accept it? (just to cover our bases?)

ie, something like:

> # How to use this image
> 
> To view the license:
> 
> ```console
> $ docker run --rm -e TS3SERVER_LICENSE=view %%IMAGE%%
> ```
> 
> To accept the license and run the server (mapping the appropriate ports to the host):
> 
> ```console
> $ docker run -p 9987:9987/udp -p 10011:10011 -p 30033:30033 -e TS3SERVER_LICENSE=accept %%IMAGE%%
> ```